SourcetrailExtension/SolutionParser/SolutionParser.cs

```diff
@@ -130,12 +130,15 @@ public void CreateCompileCommands(Project project, string solutionConfigurationN
 				bool isMakefileProject = vcProjectConfiguration.isMakefileConfiguration();
 
 				// create command objects for all applicable project items
-				foreach (EnvDTE.ProjectItem item in Utility.ProjectUtility.GetProjectItems(project))
 				{
-					CompileCommand command = CreateCompileCommand(item, commandFlags, cppStandard, cStandard, isMakefileProject);
-					if (command != null)
+					var compileCommands = Utility.ProjectUtility.GetProjectItems(project).Select(item =>
```

Comment:
Please use an explicit data type here to make the code more expressive.

Reply:
Done!